src/commands/run.ts

```diff
@@ -72,6 +72,16 @@ export default class Run extends Command {
       helpValue: "<path>",
       multiple: true,
     }),
+    plugin: Flags.string({
+      description: "[experimental] Path to a directory with JS plugins",
```

Comment:
I think we need to a add a link to documentation. This one I suppose: https://doc.fluence.dev/aqua-book/aqua-cli/plugins

Reply:
Wouldn't hurt